kitsune/wiki/jinja2/kb-ux-experiment/insecure-password-warning-firefox.html

```diff
@@ -96,12 +96,13 @@ <h2 class="logo">
           </form>
         </div>
       </div>
-
       <div id="breadcrumbs">
-        <ul class="breadcrumbs">
-          <li>
-            <div class="drop-menu">
-              <ul>
+        <div class="shadow"> </div>
+        <div class="breadcrumbs-wrap">
```

Comment:
These two lines are the only real addition. All the rest of the diff is from indentation changes.